crypto/rand/sysrand.c

```diff
@@ -105,4 +105,24 @@ int GFp_sysrand_chunk(void *out, size_t requested) {
   return 1;
 }
 
+#elif defined(__OpenBSD__) || defined(__FreeBSD__)
+
+#include <limits.h>
+#include <stdlib.h>
+
+// DEBUG:
+#include <stdio.h>
+
+const size_t GFp_sysrand_chunk_max_len = ULONG_MAX;
```

Comment:
Should `ULONG_MAX` be `SIZE_MAX` instead?

BSD's man page says the size argument is `size_t`: http://man.openbsd.org/arc4random.3.

And so does FreeBSD's:
https://www.freebsd.org/cgi/man.cgi?query=arc4random&sektion=3.